lib/robots/dor_repo/release/update_marc.rb

```diff
@@ -13,8 +13,33 @@ def initialize
         #
         # @param [String] druid -- the Druid identifier for the object to process
         def perform_work
-          logger.debug "update_marc working on #{druid}"
-          object_client.update_marc_record
+          logger.debug "update_marc handing off to dor-services-app job for update to #{druid}"
+          update_marc_record
+
+          # Since the actual update work is completed by dor-services-app in a job,
+          # workflow logging (marking step completed or failed) is done there.
+          LyberCore::ReturnState.new(status: :noop, note: 'Initiated update_marc_record API call.')
+        end
+
+        private
+
+        def update_marc_record
+          with_retries(max_tries:, handler: handler("Updating marc record for #{druid}", max_tries:),
```

Comment:
this is not integral to this work, and is an opportunistic touchup for resilience, left in because i'd added it when we were considering an earlier idea, where it'd have been more necessary.  if this adds more testing complexity than people think is worth having, i have no objection to getting rid of the `with_retries` wrapper (in which case the addition of the `retries` gem could be dropped also).

Reply:
ah, didn't realize DSC had built in retrying!  no objection at all to pulling it here.

Reply:
`with_retries` stuff removed